crowbar_framework/app/models/ceph_service.rb

```diff
@@ -137,9 +137,9 @@ def create_proposal
     end
     mon_nodes = mon_nodes.take(mon_nodes.length > 2 ? 3 : 1)
 
-    mds_node = select_nodes_for_role(nodes, "ceph-mds", "controller").first
+    mds_node = select_nodes_for_role(nodes, "ceph-mds", "storage").first
 
-    radosgw_node = select_nodes_for_role(nodes, "ceph-radosgw", "controller").first
+    radosgw_node = select_nodes_for_role(nodes, "ceph-radosgw", "storage").first
```

Comment:
The main thing the requirements are trying to say is to not mix the workload with the ceph services. Additionally, ideally, all ceph services (be they MONs, OSDs or RGWs) would be on separate hosts.

@cmurphy as you say, RGW is probably OK running on a controller; I imagine that's not going to be heavily loaded anyway. But, IMO all ceph roles should default to landing on nodes that were declared for storage use. The user can always shunt them elsewhere should they so chose.